patterns/ml-container-cache/cache_builder.tf

```diff
@@ -0,0 +1,20 @@
+module "ebs_snapshot_builder" {
+  source  = "clowdhaus/ebs-snapshot-builder/aws"
```

Comment:
if someone wants to embark on that saga of an adventure, sure 😅 - but I purposely put it here and not https://github.com/terraform-aws-modules because I suspect (hope) its usefulness declines to nil if better alternate means come to fruition